sample/java/src/main/java/com/datadog/android/sample/traces/TracesViewModel.java

```diff
@@ -22,6 +31,62 @@ public void stopAsyncOperations() {
             asyncTask.cancel(true);
             asyncTask = null;
         }
+
+        if (request != null) {
+            request.cancel(true);
+            request = null;
+        }
+    }
+
+    public static class Request extends AsyncTask<Void, Void, Request.Result> {
```

Comment:
maybe we should drop the "startAsync" example I added and only keep this now . I think they are both doing the same thing.

Reply:
Not necessarily, your startAsync creates the span on the main thread, so it's a good example of "create a custom span".